dbscripts/xml/upgrade.xml

```diff
@@ -204,6 +204,10 @@
 		<note file="docs/release-notes/README-3.4.0" />
 	</upgrade>
 
+	<upgrade minversion="3.1.0.0" maxversion="3.4.9.9">
+		<migration class="APP\migration\upgrade\v3_5_0\I8333_AddMissingForeignKeys" />
```

Comment:
Does this need to be run so early in the upgrade process? I would prefer if any 3.5 upgrades ran after all 3.4 upgrades (and so on), so that the upgrade process is linear.

Reply:
I didn't inspect how the integration between the `minversion`/`maxversion` and the list of `<upgrade>`s work, so I just added it here to not be forgotten =]

Reply:
The `minversion`/`maxversion` don't change during the list of upgrade steps, so this should go at the end.